src/System.CommandLine/LocalizableResourceString.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.CommandLine;
+using System.Globalization;
+using System.Linq;
+using System.Resources;
+
+namespace System.CommandLine
+{
+    /// <summary>
+    /// A localizable resource string that may possibly be formatted differently depending on culture.
+    /// Copied from https://github.com/dotnet/roslyn/blob/cf55f3a58e47298426fa971d3bd9d8857c746c65/src/Compilers/Core/Portable/Diagnostic/LocalizableResourceString.cs
+    /// </summary>
+    public sealed class LocalizableResourceString : LocalizableString
```

Comment:
Is there any reason these need to be in the root namespace? I'd prefer to keep it uncluttered.

Reply:
Nope, I am happy to move it. Any suggestions on a name?

Reply:
Maybe move it into the `System.CommandLine.Help` namespace so as not to introduce a new namespace just for this?